conflowgen/api/mode_of_transport_distribution_manager.py

```diff
@@ -19,7 +19,8 @@ def get_mode_of_transport_distribution(
     ) -> Dict[ModeOfTransport, Dict[ModeOfTransport, float]]:
         """
 
-        Returns: The distribution of mode of transports dependent on the vehicle the container is delivered by.
+        Returns:
+            The distribution of mode of transports dependent on the vehicle the container is delivered by.
```

Comment:
Please add an empty line between the returns statement and the following text block. The following text block is not what is returned.